storagext/lib/src/runtime/display/storage_provider.rs

```diff
@@ -87,12 +87,17 @@ impl std::fmt::Display for Event {
                 "Sectors Proven: {{ owner: {}, sectors: {:?} }}",
                 owner, sectors,
             )),
-            Event::SectorSlashed {
+            Event::SectorsSlashed {
                 owner,
-                sector_number,
+                sector_numbers,
             } => f.write_fmt(format_args!(
                 "Sector Slashed: {{ owner: {}, sector_number: {} }}",
-                owner, sector_number,
+                owner,
+                itertools::Itertools::intersperse(
+                    sector_numbers.iter().map(|sector| format!("{}", sector)),
```

Comment:
`format!("{}", sector)` implies that `type(sector): Display` which implies that `type(sector): ToString` as such you may find that the following is clearer
```suggestion
                    sector_numbers.iter().map(ToString::to_string),
```
If you like it, apply to the others